### DISCUSSION TITLE
Remote access is not possible

### DISCUSSION THREAD

**Discussion by Musaban1** [koush/scrypted#1095] [Q&A]
Hi! 
After i updated scrypted and iOS 17 on all my devices, remote access to my cameras is no longer possible. Here is the log when i try to access one of the cameras:

[HomeKit]: destination address 192.168.50.2 60844 51195
[HomeKit]: using address override 192.168.50.198
[HomeKit]: source address 192.168.50.198 34397 47847
[HomeKit]: handleStreamRequest {
[HomeKit]:   sessionID: '387c7533-b93c-44df-8fa9-34f5536f1023',
[HomeKit]:   type: 'start',
[HomeKit]:   video: {
[HomeKit]:     codec: 0,
[HomeKit]:     profile: 1,
[HomeKit]:     level: 2,
[HomeKit]:     packetizationMode: 0,
[HomeKit]:     cvoId: undefined,
[HomeKit]:     width: 1280,
[HomeKit]:     height: 720,
[HomeKit]:     fps: 30,
[HomeKit]:     pt: 99,
[HomeKit]:     ssrc: 4174140845,
[HomeKit]:     max_bit_rate: 299,
[HomeKit]:     rtcp_interval: 0.5,
[HomeKit]:     mtu: 1378
[HomeKit]:   },
[HomeKit]:   audio: {
[HomeKit]:     codec: 'OPUS',
[HomeKit]:     channel: 1,
[HomeKit]:     bit_rate: 0,
[HomeKit]:     sample_rate: 24,
[HomeKit]:     packet_time: 60,
[HomeKit]:     pt: 110,
[HomeKit]:     ssrc: 3108421909,
[HomeKit]:     max_bit_rate: 24,
[HomeKit]:     rtcp_interval: 5,
[HomeKit]:     comfort_pt: 13,
[HomeKit]:     comfortNoiseEnabled: false
[HomeKit]:   }
[HomeKit]: }
[HomeKit]: Will wait for initial RTCP packet. { isHomeHub: false, isLowBandwidth: true, hasHomeHub: false }
[HomeKit]: { isLowBandwidth: true, isWatch: false, destination: 'remote' }
[Rebroadcast Plugin]: Selected stream RTMP main.bcs
[Rebroadcast Plugin]: sending prebuffer 2000
[HomeKit]: { tool: undefined, rtpSender: 'Scrypted' }
[HomeKit]: Waiting for video RTCP packet before sending video.
[HomeKit]: Video RTCP Packet timed out. There may be a network (routing/firewall) issue preventing the Apple device sending UDP packets back to Scrypted.
[HomeKit]: Received first video RTCP packet.
[HomeKit]: rtsp/tcp h264
[HomeKit]: a/v h264 opus found h264 undefined
[HomeKit]: rtsp/tcp aac
[Rebroadcast Plugin]: RTMP main.bcs active rebroadcast clients: 1
[HomeKit]: -hide_banner -i rtsp:[REDACTED] -acodec libopus -application lowdelay -frame_duration 60 -flags +global_header -ar 24k -b:a 24k -bufsize 96k -ac 1 -payload_type 110 -ssrc 9035146 -dn -sn -vn -f rtp rtp://127.0.0.1:57441?pkt_size=400 -sdp_file pipe:4
[Rebroadcast Plugin]: Found sync frame in rtsp prebuffer.
[HomeKit]: received first video packet
[HomeKit]: scanning for idr start found: 24 7 8
[HomeKit]: idr start found: 28 5
[HomeKit]: [rtsp @ 0x7f97ed2ff700] method SETUP failed: 461 Unsupported Transport
[HomeKit]: Input #0, rtsp, from 'rtsp://127.0.0.1:55977':
[HomeKit]:   Metadata:
[HomeKit]:     title           : No Name
[HomeKit]: Duration: N/A, start: 0.000000, bitrate: N/A
[HomeKit]:   Stream #0:0: Audio: aac (LC), 16000 Hz, mono, fltp
[HomeKit]: Stream mapping:
[HomeKit]:   Stream #0:0 -> #0:0 (aac (native) -> opus (libopus))
[HomeKit]: Press [q] to stop, [?] for help
[HomeKit]: Output #0, rtp, to 'rtp://127.0.0.1:57441?pkt_size=400':
[HomeKit]:   Metadata:
[HomeKit]:     title           : No Name
[HomeKit]:     encoder         : Lavf58.76.100
[HomeKit]: video/audio detected, discarding further input
[HomeKit]: received first audio packet
[HomeKit]: handleStreamRequest { sessionID: '387c7533-b93c-44df-8fa9-34f5536f1023', type: 'stop' }
[HomeKit]: streaming session killed, duration: 30s
[Rebroadcast Plugin]: RTMP main.bcs active rebroadcast clients: 0
[HomeKit]: ffmpeg exited
[HomeKit]: HomeKit Connection 192.168.50.86

**Comment by benjamiah** [comment]
Same issue here. Ive played with everything and can't get any cameras to be viewed remotely. Anyone got it working? Im running Hikvision and reolink cameras with vanilla setup and Asus GT6000 router. Other HomeKit devices are accessible remotely.

**Comment by EntingChou** [comment]
Same issue here. Unable to remotely view the camera unless I'm on the local Wi-Fi network.

**Reply by jvic1234** [reply]
iOS 17 bug

**Comment by lucasteligioridis** [comment]
I'm getting this issue on my iPhone 14 Pro and my wife's iPhone 13 Pro on iOS 17 with an ATV on tvOS 17 using Scrypted.

Is this issue with Apple or some combination with Scrypted? I'm seeing a lot of things with iOS 17 and it seems to be an Apple issue but haven't seen anyone talking about it recently.

**Comment by yacob841** [comment]
Just to add some insight. I have the same devices, same camera, then Scrypted and Home Assistant. With Scrypted, I cannot view remotely. With Home Assistants Unify Protect plugin I can view remotely but I cannot have Apple record or Face Recognition. So as my interim solution, I have added them both, the Home Assistant one is for remote use, the Scrypted one is for local use/recording/facial recognition. Not ideal, but until one of them fixes their issues this is the best I can get to work.

**Comment by pappysven** [comment]
i've the same problem here. Tried to downgrade the video stream, but no success. I h'ave seen a post on reddit from a guy who made a workaround disabling the IGMP on his switch. 
Did anyone succeed ?

**Reply by pappysven** [reply]
update: disabling IGMP on your network switch works, just did it

**Comment by AppleCarmine** [comment]
same problem here

**Comment by Squuiid** [comment]
Same problem here. Did anyone manage to fix this?
